test/python/quantum_info/operators/symplectic/test_pauli_list.py

```diff
@@ -2058,20 +2058,55 @@ def qubitwise_commutes(left: Pauli, right: Pauli) -> bool:
 
         # checking that every input Pauli in pauli_list is in a group in the ouput
         output_labels = [pauli.to_label() for group in groups for pauli in group]
-        assert sorted(output_labels) == sorted(input_labels)
+        #     assert sorted(output_labels) == sorted(input_labels)
```

Comment:
This assertion needs returning to the test.

Reply:
I rewrote that test case into 
`self.assertListEqual(sorted(output_labels), sorted(input_labels))`
and left old code as comment-out by my carelessness.
I will delete this line of comment.